### ISSUE TITLE
Footer component will not display in storybook

### ISSUE THREAD

**Issue by annabarnes1138** [JohnCampionJr/vitesse-addons#2]
It seems to not like the `vite-plugin-vue-i18n` stuff
 
![image](https://user-images.githubusercontent.com/33356887/109219695-8a4f4400-777d-11eb-863f-56b2fba60267.png)

Steps to reproduce:

1. Clone repo
2. use npm or yarn to install dependencies
3. Change [this line](https://github.com/JohnCampionJr/vitesse-addons/blob/storybook/src/components/Footer.vue#L3) to `import { isDark, toggleDark } from '../logics'` (so that storybook can resolve that path)
4. Add a stories file for Footer @ `.storybook/stories/Footer.stories.js` (source below) 
```javascript
import Footer from '../../src/components/Footer.vue'

export default {
  component: Footer,
  title: 'Example/Footer',
};

const Template = (args) => ({
  components: { Footer },
  setup() {
    return { args };
  },
  template: '<Footer />',
});

export const Preview = Template.bind({});
```

**Comment by JohnCampionJr** [comment]
Yeah, Storybook can't bring in anything that needs Vite plugins to work properly.  I mainly integrated it to be able to test and document some stand alone components I'm building for a client.  Unless the official storybook ever integrates with vite, this is probably as good as it can get.

**Comment by ackushiw-rialtic** [comment]
I've been testing a vite + unocss + storybook integration https://github.com/ackushiw-rialtic/vite-storybook

@stecky My implementation won't fix your issue out of the box but could point you in the right direction
